src/sdk.ts

```diff
@@ -807,8 +916,8 @@ export default class SDK {
    *   console.error(`Failed saving viewer state: ${err}`);
    * });
    */
-  public setExtensionState(state: object): Promise<object> {
-    return this.client.setExtensionState(this.identifier, state);
+  public setExtensionState = <DataType = unknown, ResponseType = unknown>(state: DataType): Promise<ResponseType> => {
+    return this.client.setExtensionState<DataType, ResponseType>(this.identifier, state);
```

Comment:
You shouldn't have to define any of these with the `name = <>() =>` function format.